README.md

```diff
@@ -101,19 +100,15 @@ Options:
 -m:	Minimise output file size by not saving vertex and facet information
 ```
 
-The basic workflow is with two arguments, an input file as specified by
-[`iterate_fdtd_matrix.m`](./tdms/matlab/iteratefdtd_matrix.m), and an output
-file name to be created.
+The basic workflow is with two arguments, an input file as specified by [`iterate_fdtd_matrix.m`](./tdms/matlab/iteratefdtd_matrix.m), and an output file name to be created.
 
-You can choose two possible solver methods: either pseudo-spectral time-domain
-(PSTD, the default) or finite-difference (FDTD, with option
-`--finite-difference`).
+You can choose two possible solver methods: either pseudo-spectral time-domain (PSTD, the default) or finite-difference (FDTD, with option `--finite-difference`).
```

Comment:
#262 - The defaults have now swapped. Also the interpolation method is now in the input file too. Finally, they're no longer flags, but this should all come out in the form of merge conflicts.